crates/conformance-test-utils/src/madara.rs

```diff
@@ -272,4 +323,117 @@ mod tests {
         // Then
         assert_eq!(result, expected_output);
     }
+
+    /// This test verifies that the `genesis_set_storage_kakarot_contract_account` function
+    /// generates the correct tuples for a given Starknet address, keys, storage value, and
+    /// storage key offset.
+    #[tokio::test]
+    async fn test_genesis_set_storage_kakarot_contract_account() {
+        // Given
+        let starknet_address = *ACCOUNT_ADDRESS;
+        let key = U256::MAX;
+        let storage_variable_name = "storage_";
+        let value = U256::MAX;
+        let value_split = split_u256_into_field_elements(value);
+
+        // This is equivalent to setting the storage of Kakarot's `storage_` variable at
+        // index 2^256 - 1 to 2^256 - 1.
+        // The first storage key is for value.low.
+        // The second storage key is for value.high.
+        let expected_output = vec![
+            (
+                (
+                    starknet_address.into(),
+                    get_storage_var_address(storage_variable_name, &split_u256_into_field_elements(key))
+                        .unwrap()
+                        .into(), // offset for value.low
+                ),
+                value_split[0].into(), // value.low
+            ),
+            (
+                (
+                    starknet_address.into(),
+                    (get_storage_var_address(storage_variable_name, &split_u256_into_field_elements(key)).unwrap()
+                        + FieldElement::from(1u64))
+                    .into(), // offset for value.high
+                ),
+                value_split[1].into(), // value.high
+            ),
+        ];
+
+        // When
+        let result = genesis_set_storage_kakarot_contract_account(starknet_address, key, value).unwrap();
+        // Then
+        assert_eq!(result, expected_output);
+    }
+
+    #[tokio::test]
+    async fn test_kakarot_contract_account_storage() {
+        let starknet_test_sequencer = construct_kakarot_test_sequencer().await;
```

Comment:
when https://github.com/kkrt-labs/kakarot-rpc/pull/390 is merged, you'll be able to rebase and simplify this

Reply:
Yeah we can remove a lot of this boilerplate code 💪